4337/test/eip4337/EIP4337ModuleExisting.spec.ts

```diff
@@ -10,6 +11,8 @@ import {
 } from '../../src/utils/userOp'
 import { chainId } from '../utils/encoding'
 
+type ParseLog = Parameters<Interface['parseLog']>[0]
```

Comment:
This is an unfortunate issue where `ethers.Interface.parseLog` expects a type that is not compatible with `ethers.Log` (but the data is compatible). I proposed a fix for it here https://github.com/ethers-io/ethers.js/pull/4459.

Reply:
Fixed with module augmenting.